### PR TITLE
fix(LayoutMenuVariant): change border color of accordion area

### PR DESCRIPTION
**Pull request by saghdaey**
### Overview

Updates border color of the accordion area to navy-300 as per design request

### PR Checklist

- [X] Related to designs:
- [X] Related to JIRA ticket: [DISC-1069]
- [X] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [X] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories
NOTE: I'm seeing failures unrelated to my gamut changes in both Codecademy & Mono right now...

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

Slack conversation for reference: https://codecademy.slack.com/archives/G015E08LW3T/p1675693102223139
Screenshot since it's from a thread in a private channel:
Screen Shot 2023-02-06 at 1 58 26 PM

1. Visit storybook and see the updated divider color in the LayoutMenuVariant component
2. Visit the mono preview environment and note the updated color
3. Approve!

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/34578)  | [Monolith Env](https://pr-34578-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/1211)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-1211)   |

[DISC-1069]: https://codecademy.atlassian.net/browse/DISC-1069?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/gamut-labs/src/LayoutMenuVariant/AccordionMenu.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const StyledAccordionArea = styled(AccordionArea)`
   padding-block: ${({ theme }) => theme.spacing[16]};
   position: relative;
   left: -4px;
-  border-bottom: 1px solid ${({ theme }) => theme.colors['navy-500']};
+  border-bottom: 1px solid ${({ theme }) => theme.colors['navy-200']};
```
Comment (saghdaey) [inline packages/gamut-labs/src/LayoutMenuVariant/AccordionMenu.tsx:13]:
```suggestion
  border-bottom: 1px solid ${({ theme }) => theme.colors['navy-300']};
```

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.276-alpha.aa3cb4.0<br/>@codecademy/gamut-labs@35.5.2-alpha.aa3cb4.0<br/>@codecademy/styleguide@63.4.2-alpha.aa3cb4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63e1274f8965f107e33c5bd4--gamut-preview.netlify.app>https://63e1274f8965f107e33c5bd4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63e1274f8965f107e33c5bd4>Deploy Logs</a></p>

**Review by medhatnar — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.277-alpha.6b2518.0<br/>@codecademy/gamut-labs@35.5.3-alpha.6b2518.0<br/>@codecademy/styleguide@63.5.1-alpha.6b2518.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63e16737d0dd0d0358a38aad--gamut-preview.netlify.app>https://63e16737d0dd0d0358a38aad--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63e16737d0dd0d0358a38aad>Deploy Logs</a></p>

**Pull request merged**